lib/code-manager.js

```diff
@@ -100,17 +114,21 @@ export function getCodeToInspect() {
   return [code, cursorPosition];
 }
 
-export function getCommentStrings(scope) {
-  if (parseFloat(atom.getVersion()) <= 1.1) {
-    return store.editor.languageMode.commentStartAndEndStringsForScope(scope);
-  }
-  return store.editor.getCommentStrings(scope);
+export function getCommentStrings(
+  editor: atom$TextEditor,
+  scope: atom$ScopeDescriptor
+) {
+  // do we need this block below? No, according to package.json, if yes need types
+  // if (parseFloat(atom.getVersion()) <= 1.1) {
+  //   return editor.languageMode.commentStartAndEndStringsForScope(scope);
+  // }
+  return editor.getCommentStrings(scope);
```

Comment:
I'm OK with removing this function and use `editor.getCommentStrings(scope)` everywhere, as long as we edit the readme that we only support Atom `1.11+`. See #430.